### ISSUE TITLE
Move search input above table/listview

### ISSUE THREAD

**Issue by bjarnef** [leekelleher/umbraco-bulk-user-admin#12]
I think the search/filer input should be moved above the table/listview like in core, so it doesn't take up valuable space in listview.

Maybe it could be more consistent with Umbraco 7.4+?

Umbraco Forms does use method to check for the Umbraco version and if it can use newer components like `umb-overlay`

**Comment by leekelleher** [comment]
@bjarnef Thanks for the suggestion! 👍 

We developed this against v7.2.x, so for this release that was our target audience.  I'm happy for this to make use of newer directives in latest Umbraco... but it's not a high priority for me at this moment, at least not until I need to upgrade my customer to vLatest.  Totally happy to accept a PR from the community! 💯

**Comment by bjarnef** [comment]
@leekelleher okay, but I think if should be safe to move the search/filter input.. I submitted a PR for that in core, but can't remember if that was merged into v.7.2.x or v.7.3.x

**Comment by jamiepollock** [comment]
@leekelleher @bjarnef this seems like a big one but it'd make more sense to look into this one first before #14 or #15.

**Comment by jamiepollock** [comment]
To qualify my last comment, I mean I'd be looking into moving to using the listview which is present in Umbraco 7.4/7.5.

**Comment by jamiepollock** [comment]
@leekelleher @bjarnef I've looked into utilising umb-table seems like a starter.

My current implementation is a step backward as we've lost the UI for searching but we've gained a uniform way of utilising the umb-table view which reduced the dashboard markup & moved the JS code slightly.

I'm going to keep plugging away but I'm fairly hopefully with this new direction even if at first it meant a few internal changes.
